js/language_server/src/tests/integration.test.ts

```diff
@@ -9,7 +9,7 @@ const EXPECTED_CAPABILITIES = {
     codeActionProvider: true,
     hoverProvider: true,
     textDocumentSync: {
-      change: 1,
+      change: 2,
```

Comment:
2? why not 3? Is it an enum? Maybe a comment explaining the possible values here?